tests/virtiofs/datavolume.go

```diff
@@ -246,22 +246,8 @@ var _ = Describe("[sig-storage] virtiofs", decorators.SigStorage, func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(strings.Trim(podVirtioFsFileExist, "\n")).To(Equal("exist"))
 			By("Finding virt-launcher pod")
-			var virtlauncherPod *k8sv1.Pod
-			Eventually(func() *k8sv1.Pod {
-				podList, err := virtClient.CoreV1().Pods(vmi.Namespace).List(context.Background(), metav1.ListOptions{})
-				if err != nil {
-					return nil
-				}
-				for _, pod := range podList.Items {
-					for _, ownerRef := range pod.GetOwnerReferences() {
-						if ownerRef.UID == vmi.GetUID() {
-							virtlauncherPod = &pod
-							break
-						}
-					}
-				}
-				return virtlauncherPod
-			}, 30*time.Second, 1*time.Second).ShouldNot(BeNil())
+			virtlauncherPod, err := libpod.GetPodByVirtualMachineInstance(vmi, testsuite.GetTestNamespace(vmi))
```

Comment:
Should this generate some flakiness? asking

Reply:
Do you mean `Expect(virtlauncherPod.Spec.Containers).To(HaveLen(4))`?
It generates failures on platforms that opinionate kubevirt (so for example hco disables the guest log container)

Reply:
Nope, sorry! I mean, previously we had an `Eventually` cycle, while now we have just one shot. So I was wondering if this can cause some sort of flakiness. I totally agree about the removal of the container length

Reply:
ah okay I think we are good since it's called after successful VM start (launcher must exist at this point)

Reply:
Fair enough, sorry for being lazy!
/lgtm